access-token-management/samples/WebJarJwt/ClientAssertionService.cs

```diff
@@ -59,6 +59,11 @@ public ClientAssertionService(
                 { JwtClaimTypes.JwtId, Guid.NewGuid().ToString() },
                 { JwtClaimTypes.Subject, config.ClientId! },
                 { JwtClaimTypes.IssuedAt, DateTime.UtcNow.ToEpochTime() }
+            },
```

Comment:
We need to also change the audience above (Line 53) to be the issuer aka the authority.

Reply:
Following up in a separate PR, as this merged as I was writing my comment.